.circleci/config.yml

```diff
@@ -197,6 +197,37 @@ jobs:
           -X POST https://slack.com/api/chat.postMessage  
 
     - post_steps
+  e2e-preview:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.41.2-jammy
+    steps:
+      - checkout
+      - run: npm i -D @playwright/test
+      - run: npx playwright install chrome
+      - run:
+          name: Wait for preview to be up, poll every 10 seconds
+          command: |
+            GETURL="https://preview-emx2-pr-${CIRCLE_PULL_REQUEST##*/}.dev.molgenis.org/"
+            while true; 
```

Comment:
I see your point. My 'timeout' suggestion did not work? Because I agree this code is a bit much (although nice).

Reply:
no not as is , suggest to try this and revert to fixed time out if it turn out to be flacky